From 56503ae7d7384529d5a638019990642bae514c15 Mon Sep 17 00:00:00 2001
From: Neta Zur <neta@marvell.com>
Date: Tue, 6 Feb 2018 14:37:40 +0200
Subject: [PATCH 2215/2241] fix: dts: mvebu: cp110: update names of cp0 and cp1

Update cp110-0 to cp0 and cp110-1 to cp1
Name change was added in commit -
dts: cp110: remove duplicated files for multiple cp110 instances

Change-Id: I477999cebc55a7a6c19fe43074c60adb825fbca3
Signed-off-by: Neta Zur <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/50115
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts | 2 +-
 arch/arm64/boot/dts/marvell/armada-8020.dtsi      | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
index 4369f5a..3cbbc24 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd.dts
@@ -47,7 +47,7 @@
 	compatible = "marvell,armada3900-db-vd", "marvell,armada7040",
 		     "marvell,armada-ap806-quad", "marvell,armada-ap806";
 
-	cp110-0 {
+	cp0 {
 		config-space {
 			pinctrl@440000 {
 				cp0_xhci0_vbus_pins: cp0-xhci0-vbus-pins {
diff --git a/arch/arm64/boot/dts/marvell/armada-8020.dtsi b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
index 9fbcdf4..cdb49a8 100644
--- a/arch/arm64/boot/dts/marvell/armada-8020.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8020.dtsi
@@ -189,7 +189,7 @@
 		};
 	};
 
-	cp110-0 {
+	cp0 {
 		config-space {
 			pinctrl@440000 {
 				compatible = "marvell,a80x0-cp0-pinctrl";
@@ -246,7 +246,7 @@
 		};
 	};
 
-	cp110-1 {
+	cp1 {
 		config-space {
 			pinctrl@440000 {
 				compatible = "marvell,a80x0-cp1-pinctrl";
-- 
2.7.4

